roaringbitmap/src/main/java/org/roaringbitmap/RoaringArray.java

```diff
@@ -345,6 +355,287 @@ public void deserialize(DataInput in) throws IOException {
     }
   }
 
+  /**
+   * Deserialize.
+   *
+   * @param in the DataInput stream
+   * @param buffer a byte[] used to buffer DataInput reading
+   * @throws IOException Signals that an I/O exception has occurred.
+   * @throws InvalidRoaringFormat if a Roaring Bitmap cookie
+   *             is missing.
+   */
+  public void deserialize(DataInput in, byte[] buffer) throws IOException {
+    if (buffer != null && buffer.length == 0) {
+      // Get ride of this useless buffer
```

Comment:
I think that the correct spelling is "get rid of"?

Reply:
Fixed